website_compassion/template/my_account_donations.xml

```diff
@@ -261,5 +289,19 @@
                 <a href="https://compassion.ch/donations/" class="btn btn-primary" target="_blank">Make a donation</a>
             </div>
         </template>
+        <template id="my_donations_upgrade" name="upgrade">
+            <t t-set="selected">donation</t>
+            <t t-set="title">My financials</t>
+            <t t-call="website_compassion.my_account_layout">
+               <center> Comfirm upgrade to 50 CHF ? </center>
```

Comment:
Same as above
If selected sponsorship < 42 CHF the upgrade wouldn't be all the way to 50 (with current implementation).
Maybe a bit confusing for the end user ?